### DISCUSSION TITLE
[day1] gazebo 및 rviz2 실행시 나오는 터미널 오류

### DISCUSSION THREAD

**Discussion by Kim-JeongHan** [Road-Balance/gcamp_ros2_basic#53] [Q&A] [answered]
사용 os: ubuntu 20.04
사용 ros 버전: ros2 foxy
![image](https://user-images.githubusercontent.com/98142496/179153774-608061bc-e941-4085-b37f-17bae705651c.png)

gazebo는 정상 실행이 되는 것 같은데 그림과 같은 오류가 발생했다고 뜨는데 원래 그런건가요. 아니면 무시하고 넘어가도 될만한 오류일까요?

![image](https://user-images.githubusercontent.com/98142496/179153938-03065c43-b29a-4ece-bbfe-cd211f62be18.png)
rviz2실행시에도 그림과 같은 오류가 발생하는데 실행할때에는 문제가 생기지 않습니다! ROS1을 사용할때는 이런식의 오류도 발생하지 않았던 것으로 기억해서 고칠수 있을까요?

**Comment by kimsooyoung** [comment, marked as answer]
무시하셔도 좋습니다. 해당 로그는 저 또한 매번 발생하고 있습니다.

![image](https://user-images.githubusercontent.com/12381733/179442036-a8a25e3a-e00b-44ae-b497-fb3323bc760d.png)
